### PR TITLE
create show food header

### PR DESCRIPTION
**Pull request by tuanlinhtl17**

### CODE REVIEW

**Comment by dinhhieu165** [comment]
OK

File: app/views/foods/_header.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      <div class="col-md-5">
+        <img src="<%= food.img_url %>" alt="<%= food.name %>" title="<%= food.name %>">
+      </div>
+      <div class="col-md-7">
+        <h1><%= food.name %></h1>
+        <p><%= "#{t "views.foods.show.header.category"}: " %><%= link_to category.name, category %></p>
+        <%= render "foods/star_rating" %>
+        <p><%= "#{food.rating_avg} / #{food.rate_count} " %><%= t "views.foods.show.header.rate" %></p>
+        <p><%= t "views.foods.show.header.price" %>: $<%= food.price %></p>
+        <p><%= t "views.foods.show.header.chef" %>: <%= link_to User.find(food.user_id).name, User.find(food.user_id) %></p>
```
Comment (chuanhtuan) [inline app/views/foods/_header.html.erb]: food.user

File: app/views/foods/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        <hr>
+        <h5><%= t "views.foods.show.header.delivery_fee" %>: 5,000 đ/km</h5>
```
Comment (chuanhtuan) [inline app/views/foods/_header.html.erb]: gem setting

File: app/views/foods/show.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<%= render partial: "foods/header", locals: {food: @food, category: FoodCategory.find(@food.food_category_id)} %>
```
Comment (chuanhtuan) [inline app/views/foods/show.html.erb]: category: @food.food_category

**Pull request merged**